### DISCUSSION TITLE
Is there a ui that can handle i64 in responses?

### DISCUSSION THREAD

**Discussion by DenuxPlays** [juhaku/utoipa#966] [Q&A]
Hey,

I used swagger-ui until now but I found out that swagger cannot handle integers that are larger than 32 bit.
The same with scalar.

Is there a ui that support them + have a try it out feature?

**Comment by juhaku** [comment]
I believe this should be possible as `i64` is so called long in other languages, but bigger than that might have some undefined behavior since I am not sure how well JavaScript would handle them. 

However if there are issues with `i64`, it sounds like a bug somewhere but I might be mistaken.

**Reply by DenuxPlays** [reply]
No there is not.

Its a bug in every js application because BigInt (what could represent i64 and u64) cannot be serialized to json using JSON.stringify()
Thats why they can not/ dont want to (or what ever) implement i64 support in swagger-ui and scalar for example.

**Reply by juhaku** [reply]
Wow, Darn JavaScript (good for nothing but prototyping). Good finding though. Another reason not to use it if not necessary for the usecase.
